Client/src/Users.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
As we learned pairing up, name spacing is tricky. The Udemy course made a big mistake using an index.js file for QueryRenderer. I know this took an hour to debug and we never want to make this mistake again! I put the info in th README